docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,282 @@
+Open MPI v5.0.x series
+======================
+
+This file contains all the NEWS updates for the Open MPI v5.0.x
+series, in reverse chronological order.
+
+Open MPI version 5.0.0rc3
+-------------------------
+:Date: 08 March 2022
+
+.. admonition:: MPIR API has been removed
+   :class: warning
+
+   As was announced in summer 2017, Open MPI has removed support of
+   MPIR-based tools beginning with the release of Open MPI v5.0.0.
+
+   The new PRRTE based runtime environment supports PMIx-tools API
+   instead of the legacy MPIR API for debugging parallel jobs.
+
+   see https://github.com/openpmix/mpir-to-pmix-guide for more
+   information
+
+
+.. admonition:: zlib is suggested for better user experience
+   :class: note
+
+   PMIx will optionally use zlib to compress large data streams.
+   This may result in shorter-than-normal startup times and
+   smaller memory footprints.  It is recommended to install zlib
+   and zlib-devel for a better user experience.
```

Comment:
```suggestion
   PMIx will optionally use zlib to compress large data streams.
   This may result in faster startup times and
   smaller memory footprints (compared to not using compression).  The Open MPI community recommends building zlib support with PMIx, regardless of whether you are using an externally-installed PMIx or the PMIx that is installed with Open MPI.
```

Re-wrap those lines as appropriate.